influxql/engine.go

```diff
@@ -496,11 +496,19 @@ func (m *MapReduceJob) processAggregate(c *Call, reduceFunc ReduceFunc, resultVa
 		}
 	}
 
+	firstInterval := m.interval
+	if !m.stmt.IsRawQuery {
```

Comment:
@pauldix I removed this.  It should only get called for aggregates.  Tests are still all passing.